bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/config/config2.xml

```diff
@@ -96,10 +96,6 @@
 			<description>@text/thing-type.config.shelly.deviceIp.description</description>
 			<context>network-address</context>
 		</parameter>
-		<parameter name="userId" type="text" required="false">
-			<label>@text/thing-type.config.shelly.userId.label</label>
-			<description>@text/thing-type.config.shelly.userId.description</description>
```

Comment:
Please remove both entries from the i18n file if they are no more used.

Reply:
Please comment before marking as resolved without changing anything.
Why the corresponding entries in the i18n file are not removed if you remove the config parameter ?

Reply:
hmm, could it be that writing a comment and clicking Resolve doesn't save the comment? I'm sure that I provided feedback, but this would explain the confusion, because I gave feedback on all of them (like this one), but doesn't see it now. Super sorry if this was my handling fault.

Reply:
In fact Gen1 requires userId, but Gen2 doesn't so I removed it in between in config2.xml like requested, but obviously my comment got lost.

Reply:
Ok, I checked how it is done to understand. You made reference to existing i18n entries using `@test`.
And yes, the entries for userId are still usefull for gen1.

Reply:
> hmm, could it be that writing a comment and clicking Resolve doesn't save the comment?

Yes, you have to post your comment by clicking "Comment", and then you can resolve it afterwards as an additional action.